asab/config.py

```diff
@@ -157,8 +172,39 @@ def _load(self):
 
 		self.add_defaults(ConfigParser._default_values)
 
+		# Obtain password section
+		self._passwords = dict(self.items("passwords"))
+
+		# The password section is truly loaded after all other sections are interpolated
+		# Iterate through all configuration options and parse passwords
+		for section in self._sections.keys():
```

Comment:
I'm actually not sure about this automated expansion ... I would pass that to the individual user (e.g. MySql connection will go to the passwords itself).
Or there is some kind of expansion in Python ConfigParser already ... and I propose to use that one then.

This doesn't seems right ...